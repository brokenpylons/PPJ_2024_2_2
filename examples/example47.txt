(x^3 - 7) / (y^(1/2)) * (2 * (z + 1) / w)^0.8
