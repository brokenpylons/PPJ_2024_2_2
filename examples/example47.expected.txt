lparen("(") variable("x") pow("^") real("3") minus("-") real("7") rparen(")") divides("/") lparen("(") variable("y") pow("^") lparen("(") real("1") divides("/") real("2") rparen(")") rparen(")") times("*") lparen("(") real("2") times("*") lparen("(") variable("z") plus("+") real("1") rparen(")") divides("/") variable("w") rparen(")") pow("^") real("0.8") 
